manual/python/bi_changes.md

```diff
@@ -0,0 +1,223 @@
+| callable.\_threaded                       | callable.\_\_release\_gil\_\_   |
+
+- New PyROOT does not parse command line arguments by default anymore. For example, when running
+`python my_script.py -b`, the `-b` argument will not be parsed by PyROOT, and therefore the
+batch mode will not be activated. If the user wants to enable the PyROOT argument parsing again,
+they can do so by starting their Python script with:
+~~~ {.python}
+import ROOT
+ROOT.PyConfig.IgnoreCommandLineOptions = False
+~~~
+
+- In new PyROOT, `addressof` should be used to retrieve the address of fields in a struct,
+for example:
+~~~ {.python}
+> ROOT.gInterpreter.Declare("""
+struct MyStruct {
+  int a;
+  float b;
+};
+""")
+
+> s = ROOT.MyStruct()
+
+> ROOT.addressof(s, 'a')
+94015521402096L
+
+> ROOT.addressof(s, 'b')
+94015521402100L
+
+> ROOT.addressof(s)
+94015521402096L
+~~~
+In old PyROOT, `AddressOf` could be used for that purpose too, but its behaviour was inconsistent.
+`AddressOf(o)` returned a buffer whose first position contained the address of object `o`, but
+`Address(o, 'field')` returned a buffer whose address was the address of the field, instead
+of such address being contained in the first position of the buffer.
+Note that, in the new PyROOT, `AddressOf(o)` can still be invoked, and it still returns a buffer
+whose first position contains the address of object `o`.
+
+- In old PyROOT, there were two C++ classes called `TPyMultiGenFunction` and `TPyMultiGradFunction`
+which inherited from `ROOT::Math::IMultiGenFunction` and `ROOT::Math::IMultiGradFunction`, respectively.
+The purpose of these classes was to serve as a base class for Python classes that wanted to inherit
+from the ROOT::Math classes. This allowed to define Python functions that could be used for fitting
+in Fit::Fitter.
+In the new PyROOT, `TPyMultiGenFunction` and `TPyMultiGradFunction` do not exist anymore, since their
+functionality is automatically provided by new cppyy: when a Python class inherits from a C++ class,
+a wrapper C++ class is automatically generated. That wrapper class will redirect any call from C++
+to the methods implemented by the Python class. Therefore, the user can make their Python function
+classes inherit directly from the ROOT::Math C++ classes, for example:
+~~~ {.python}
+import ROOT
+
+from array import array
+
+class MyMultiGenFCN( ROOT.Math.IMultiGenFunction ):
+    def NDim( self ):
+        return 1
+
+    def DoEval( self, x ):
+        return (x[0] - 42) * (x[0] - 42)
+
+    def Clone( self ):
+        x = MyMultiGenFCN()
+        ROOT.SetOwnership(x, False)
+        return x
+
+def main():
+    fitter = ROOT.Fit.Fitter()
+    myMultiGenFCN = MyMultiGenFCN()
+    params = array('d', [1.])
+    fitter.FitFCN(myMultiGenFCN, params)
+    fitter.Result().Print(ROOT.std.cout, True)
+
+if __name__ == '__main__':
+    main()
+~~~
+
+- Inheritance of Python classes from C++ classes is not working in some cases. This is described
```

Comment:
this part is completely messed up, it's all displayed in a terminal screen style